.github/readmes/experimental/typescript/graphql-auth/README.md

```diff
@@ -1,17 +1,17 @@
 # GraphQL Server with Authentication & Permissions
 
-This example shows how to implement a **GraphQL server with an email-password-based authentication workflow and authentication rules**, based on Prisma, [graphql-yoga](https://github.com/prisma/graphql-yoga), [graphql-shield](https://github.com/maticzav/graphql-shield) & [GraphQL Nexus](https://nexus.js.org/).
+This example shows how to implement a **GraphQL server with an email-password-based authentication workflow and authentication rules**, based on Prisma, [graphql-yoga](https://github.com/prisma/graphql-yoga), [graphql-shield](https://github.com/maticzav/graphql-shield) & [Nexus Schema](https://nxs.li/components/standalone/schema).
 
-__INLINE(../../_setup-1.md)__
+**INLINE(../../\_setup-1.md)**
```

Comment:
This needs to be undone as well @jasonkuhrt

Reply:
It would be helpful if things were aligned with prettier. As it stands I'll figure out how to disable autoformatting in my editor for prettier.